CODEOWNERS

```diff
@@ -0,0 +1,59 @@
+# See also:
+# - https://github.com/blog/2392-introducing-code-owners
+# - https://help.github.com/articles/about-codeowners/
+
+# Each line is a file pattern followed by one or more owners (sorted alphabetically).
+# Please feel free to add yourself to a module or create a new mapping.
+# Ideally each module should have two or more owners.
+
+# Code owners are automatically requested for review
+# when someone opens a pull request that modifies code that they own.
+# Later matches take precedence.
+
+src/checkedint.d @redstar @andralex @JackStouffer
+
+src/core/atomic.d @WalterBright
+src/core/attribute.d @jaoc-carlborg
+src/core/bitop.d @schveiguy @tsbockman @mathias-lang-sociomantic @Geod24
+src/core/cpuid.d @WalterBright @mleise @JackStouffer
+src/core/demangle.d @WalterBright @MartinNowak @rainers @ibuclaw
+src/core/exception.d @MartinNowak @WalterBright @jmdavis @CyberShadow
+src/core/internal @MartinNowak @schveiguy
+src/core/math.d @ibuclaw @redstar
+src/core/runtime.d @MartinNowak @Abscissa
+src/core/simd.d @WalterBright @MartinNowak
+src/core/stdc/* @schveiguy @ibuclaw
+src/core/stdcpp/* @WalterBright @Darredevil
+src/core/sync/* @MartinNowak @mathias-lang-sociomantic @Geod24 @WalterBright @ZombineDev
+src/core/sys/bionic/* @joakim-noah
+src/core/sys/darwin/* @klickverbot
```

Comment:
I think the names for `src/core/sys/darwin/*` and `src/core/sys/osx/*` should be the same.